### PR TITLE
feat: itsg33 issues gen

### PR DESCRIPTION
**Pull request by ascheid**
# Summary | Résumé

Python code that receives a csv with ITSG33 Controls and creates issues so it can be track in a relevant way by projects.

### CODE REVIEW

File: tools/itsg33-issue-generator/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+requests
+pytest
+unittest
```
Comment (patheard) [inline tools/itsg33-issue-generator/requirements.txt]: Could you pin these to their latest versions when you get a chance.

File: tools/itsg33-issue-generator/script.py [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+import string
+import logging
+from enum import Enum
+
+"""
+env vars:
+REPO = owner/repo
+GITHUB_TOKEN = github token to create issues
+CSV_FILE = path to csv file
+DEBUG = True/False to print debug info
```
Comment (patheard) [inline tools/itsg33-issue-generator/script.py]: I like this block explaining the vars.  Can you swap the `DEBUG` one for a `LOG_LEVEL` var and default it to whatever logging level you want if it isn't specified.

File: tools/itsg33-issue-generator/script.py [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+    GENERAL_GUIDE = 14
+    SUGGESTED_PRIORITY = 15
+    SUGGESTED_PLACEHOLDER_VALUES = 16
+    PROFILE_SPECIFIC_NOTES = 17
+
+
+"""
+Set logging level
+"""
+logging.basicConfig(level=logging.ERROR, format='%(asctime)s :%(levelname)s:%(funcName)s:%(lineno)d %(message)s', datefmt='%d-%b-%y %H:%M:%S')
```
Comment (patheard) [inline tools/itsg33-issue-generator/script.py]: You can then use your fancy new `LOG_LEVEL` global var here:
```suggestion
logging.basicConfig(level=LOG_LEVEL, format='%(asctime)s :%(levelname)s:%(funcName)s:%(lineno)d %(message)s', datefmt='%d-%b-%y %H:%M:%S')
```

File: tools/itsg33-issue-generator/script.py [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+
+
+def main():
+
```
Comment (patheard) [inline tools/itsg33-issue-generator/script.py]: nit: missing method comment

File: tools/itsg33-issue-generator/requirements.txt (hunk 1 above)
Comment (ascheid) [reply tools/itsg33-issue-generator/requirements.txt]: Done!

File: tools/itsg33-issue-generator/script.py (hunk 2 above)
Comment (ascheid) [reply tools/itsg33-issue-generator/script.py]: Done!

File: tools/itsg33-issue-generator/script.py (hunk 3 above)
Comment (ascheid) [reply tools/itsg33-issue-generator/script.py]: Done!

File: tools/itsg33-issue-generator/script.py (hunk 4 above)
Comment (ascheid) [reply tools/itsg33-issue-generator/script.py]: Done!

File: tools/itsg33-issue-generator/script.py [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+env vars:
+REPO = owner/repo
+GITHUB_TOKEN = github token to create issues
+CSV_FILE = path to csv file
+LOG_LEVEL = If exists, set logging level to this. Otherwise, set to INFO
+"""
+REPO = os.getenv("REPO")
+GITHUB_TOKEN = os.getenv("GITHUB_TOKEN")
+CSV_FILE = "controls.csv"
+LOG_LEVEL = os.getenv("LOG_LEVEL") if os.getenv("LOG_LEVEL") else logging.INFO
```
Comment (patheard) [inline tools/itsg33-issue-generator/script.py]: A trick you can use here to make it even more concise is to use the `getenv` default value that it'll fallback to if `LOG_LEVEL` isn't set:
```suggestion
LOG_LEVEL = os.getenv("LOG_LEVEL", "INFO")
```

Comment (ascheid) [reply tools/itsg33-issue-generator/script.py]: Implemented the suggestion with the use of provided constant for the log level.

**Review by patheard — APPROVED** [review]

**Pull request merged**